blocks/lead-art-block/index.story.jsx

```diff
@@ -0,0 +1,506 @@
+      '105x59': 'qVVEwBb69ga8csTW9xP7Es_IFRw=filters:format(jpg):quality(70)/',
+      '158x89': '7DKcZyHOaPJRkcyUDQ1bm5fAJfA=filters:format(jpg):quality(70)/',
+      '274x154': 'v1lLozX5SBR7lZr-aNSbts9uUTg=filters:format(jpg):quality(70)/',
+      '377x212': 'koInA_YEnB30YCI6DXWvllogoOE=filters:format(jpg):quality(70)/',
+      '400x225': 'uBrrOxONDb6BETNtysXPjtqZ09U=filters:format(jpg):quality(70)/',
+      '600x338': 'FcG7qqvzw7yUWmCLfjlLXGktAL0=filters:format(jpg):quality(70)/',
+      '768x432': 'YzKKCuCBVaHkDvvHf2fRxaKq420=filters:format(jpg):quality(70)/',
+      '800x450': 'TU1-XiQjCcY0rViV34XW4wzdbVM=filters:format(jpg):quality(70)/',
+      '1024x576': 'MBeZVprh3unBIe44ZTzR3aik_Ao=filters:format(jpg):quality(70)/',
+      '1440x810': 'DkH9txoRinjt4ozypLuG5JIg9Nw=filters:format(jpg):quality(70)/',
+      '1600x900': '8GJwWEBtsOme7kCFaWAUn2rGH_E=filters:format(jpg):quality(70)/',
+      '84x84': '2DW1k9NGgGfDLsA1RV8b4CDoLtg=filters:format(jpg):quality(70)/',
+      '105x105': 'n0QP-cEW65SLSRkl70UKF80IQCc=filters:format(jpg):quality(70)/',
+      '158x158': 'eoHGAh8LcdGG2JF_djsXLNLZB6A=filters:format(jpg):quality(70)/',
+      '274x274': 'iTGqN49xCxhIuEC5LWXrlJfyhcw=filters:format(jpg):quality(70)/',
+      '377x377': 'BEEHSP93TU8IYANDyOUiOXJExZo=filters:format(jpg):quality(70)/',
+      '400x400': 'QdJm_OpVStiUC9CMrmXsVTJx6nA=filters:format(jpg):quality(70)/',
+      '600x600': 'SzlxQtqGicQi6zsyIgH5YEDSGjM=filters:format(jpg):quality(70)/',
+      '768x768': 'EAkab8QloV6gGpBeef2_yOPBayg=filters:format(jpg):quality(70)/',
+      '800x800': 'yRCAQ4ClgrLJF9uxmxrWBQhvSdo=filters:format(jpg):quality(70)/',
+      '1024x1024': 'DhEwj1PWT_1eXtio-mvg-6x0C0I=filters:format(jpg):quality(70)/',
+      '1440x1440': 'hDja6DXewa2TrorD4hRG4RDGFso=filters:format(jpg):quality(70)/',
+      '1600x1600': 'wbcW7lJyK8-4-GQtZGrkYurh6d4=filters:format(jpg):quality(70)/',
+    },
+  }],
+};
+
+const leadArtVideo = {
+  type: 'video',
+  credits: {
+    affiliation: [{
+      name: 'Affiliation Name',
+    }],
+    by: [{
+      type: 'author',
+      name: 'Author Name',
+      org: 'Author Org',
+      slug: '',
+    }],
+  },
+  subtitles: {},
+  embed_html: '<div class="powa" id="powa-ba52f779-47be-46b9-8bd5-58dcb4318101" data-org="corecomponents" data-env="prod" data-uuid="ba52f779-47be-46b9-8bd5-58dcb4318101" data-aspect-ratio="0.562" data-api="prod"><script src="//d2w3jw6424abwq.cloudfront.net/prod/powaBoot.js?org=corecomponents"></script></div>',
+  caption: 'Sample caption',
+};
+
+export const gallery = () => {
+  const globalContent = {
+    arcSite: 'story-book',
+    promo_items: {
+      lead_art: leadArtGallery,
+    },
+  };
+  return (
+    <LeadArt globalContent={globalContent} />
+  );
+};
+
+export const image = () => {
+  const globalContent = {
+    arcSite: 'story-book',
+    promo_items: {
+      lead_art: leadArtImage,
+    },
+  };
+  return (
+    <LeadArt globalContent={globalContent} />
```

Comment:
The image is not showing within Chromatic - https://www.chromatic.com/test?appId=5f91e4721ccaba0022a10782&id=61032846d790750039b0282d

Reply:
Changed out which image this uses.  It was working but now I'm getting an error...